vite.config.ts

```diff
@@ -0,0 +1,18 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+/// <reference types="vitest" />
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+    test: {
+        globals: true,
+        coverage: {
+            provider: 'c8',
+            reporter: ['text', 'html']
```

Comment:
Just a minor comment: We're only looking for coverage in our core framework package (as of now). Also, we're excluding generated files.

```suggestion
            provider: 'c8',
            reporter: ['text', 'html'],
            include: ['packages/langium/src'],
            exclude: ['**/generated']
```